src/dayview.css

```diff
@@ -87,14 +87,36 @@
   display: block !important;
 }
 
+.dayview-category-header {
+  position: relative;
+  font-size: 14px;
+}
+
+.dayview-category-header-item {
+  font-weight: normal;
+  height: 37px;
+  border-left: 1px solid #ddd;
```

Comment:
have you tested if the border-left works in RTL mode? maybe you don't need it as you have table-bordered specified?